sale_blanket_order_line_position/models/ir_actions_report.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    def render_qweb_pdf(self, res_ids=None, data=None):
+        self.blanket_recompute_positions(res_ids)
```

Comment:
Why do you need this? The order of the lines must be always the correct one before printing.